spec/amp-versioning-policy.md

```diff
@@ -0,0 +1,40 @@
+# AMP versioning policy
+
+## Basics
+
+- We use [semver](http://semver.org/) terminology, but see below for changes from simple semver.
+- The AMP JS library is distributed via its major version only. The major version is expected to change rarely if ever (on the order of less than once per year).
+- Each AMP extension is versioned independently. Publishers of AMP documents can choose the major and minor version of an extension. They cannot choose the patch version.
+- AMP is **released** about once per weak and each release has a unique version number. With each release, AMP pages are rendered with the latest code for the respective major/minor versions of the AMP JS library and extensions.
+
+## Goals
+
+- Reduce overall JS URL entropy. AMP aims to provide as few versions of the JS library and extensions as possible to improve cache hit rates for these requests.
+- Avoid breaking existing pages.
+- Keep all users of AMP on secure versions of the library.
```

Comment:
How do we achieve this if the publisher has been pinned to an old version and a major change was required for a security fix